config/software/datadog-agent-integrations.rb

```diff
@@ -0,0 +1,100 @@
+require './lib/ostools.rb'
+
+name 'datadog-agent-integrations'
+
+dependency 'pip'
+
+relative_path 'integrations-core'
+
+env = {
+  "PATH" => "#{install_dir}/embedded/bin:#{ENV['PATH']}",
+}
+
+# The only integrations that will be packaged with the agent
+# are the ones that are officiallly supported.
+local_integrations_core_repo = ENV['LOCAL_INTEGRATIONS_CORE_REPO']
+if local_integrations_core_repo.nil? || local_integrations_core_repo.empty?
+  source git: 'https://github.com/DataDog/integrations-core.git'
+else
+  # For local development
+  source path: ENV['LOCAL_INTEGRATIONS_CORE_REPO']
+end
+
+integrations_core_branch = ENV['INTEGRATION_CORE_BRANCH']
+if integrations_core_branch.nil? || integrations_core_branch.empty?
+  default_version 'master'
+else
+  default_version integrations_core_branch
+end
+
+build do
+  if ohai['platform'] == "windows"
+    integrations_dir = project_dir
+  else
+    integrations_dir = project_dir
+  end
+
+  mkdir  "#{install_dir}/agent/"
+  # Agent code
+  mkdir "#{install_dir}/agent/checks.d"
+
+  # Grab all the checks
+  checks = Dir.glob("#{integrations_dir}/*/")
+  # logging.info('datadog-agent-integrations') { checks }
+  # logging.info('datadog-agent-integrations') { Dir.glob("*") }
+
+  # loop through them
+  checks.each do |check|
+    # Only use the parts of the filename we need
+    check.slice! "#{integrations_dir}/"
+    check.slice! "/"
+
+    # Copy the checks over
+    if File.exists? "#{integrations_dir}/#{check}/check.py"
```

Comment:
Shouldn't we do the opposite? I.e. if `check.py` doesn't exist, log an error and skip (or fail)?